Source/Core/DolphinTool/ToolMain.cpp

```diff
@@ -1,9 +1,8 @@
-// Copyright 2021 Dolphin Emulator Project
+// Copyright 2024 Dolphin Emulator Project
```

Comment:
Please don't update the copyright on existing files.

Reply:
Reverted!